index.html

```diff
@@ -28,34 +29,34 @@
       
       <section id="port-without-header">
          <div class="port-contenido">
```

Comment:
I suggest you fix the margins of the headline section. Currently your content seems squeezed yet you have a lot of empty space left.

- [ ] This is your design
<img width="519" alt="Screenshot 2022-09-24 at 11 17 27" src="https://user-images.githubusercontent.com/48546992/192088054-0cd6d53e-5048-43e2-a4b1-95e5e11ff275.png">

- [ ] This is the figma design 
<img width="268" alt="Screenshot 2022-09-24 at 11 14 00" src="https://user-images.githubusercontent.com/48546992/192088021-7dd19a64-c047-47a4-93db-c44537940137.png">